### DISCUSSION TITLE
[v3] Livewire page component layout view not found if a component contains nested component

### DISCUSSION THREAD

**Discussion by N1ebieski** [livewire/livewire#6281] [7. Bugs (archived)]
### Livewire version

3.0.0-beta.8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a Full Page Component class:

```php
<?php

namespace App\Livewire\Components;

use Livewire\Component;
use Livewire\Attributes\Layout;

class Hello extends Component
{
    #[Layout('components.layout.layout')]
    public function render()
    {
        return <<<'HTML'
            <div>
                <h1>Hello</h1>
                <livewire:nested />
            </div>
        HTML;
    }
}
```

2. Create a Nested Component class:

```php
<?php

namespace App\Livewire\Components;

use Livewire\Component;

class Nested extends Component
{
    public function render()
    {
        return <<<'HTML'
            <div>Nested Component</div>
        HTML;
    }
}
```

3. Create a resources/views/components/layout/layout.blade.php file as in the [documentation](https://livewire.laravel.com/docs/components#layout-files)

4. Add the route:

```php
Route::get('/hello', \App\Livewire\Components\Hello::class)
```

4. Go to the /hello - you'll get error:

Livewire \ Features \ SupportPageComponents \ MissingLayoutException

Livewire page component layout view not found: [components.layout.app]

### Additional information

The error will disappear if you remove the nested component or create a layout file in the path given in config/livewire.php (default: components.layout.app)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by riasvdv** [comment]
Added a failing test here as we ran into this one as well https://github.com/livewire/livewire/pull/6286

**Reply by calebporzio** [reply]
Fixed! good catch
